libafl/src/schedulers/accounting.rs

```diff
@@ -263,7 +280,16 @@ where
     /// Creates a new [`CoverageAccountingScheduler`] that wraps a `base` [`Scheduler`]
     /// and has a default probability to skip non-faved [`Testcase`]s of [`DEFAULT_SKIP_NON_FAVORED_PROB`].
     pub fn new(state: &mut S, base: CS, accounting_map: &'a [u32]) -> Self {
-        state.add_metadata(TopAccountingMetadata::new(accounting_map.len()));
+        match state.metadata().get::<TopAccountingMetadata>() {
+            Some(meta) => {
+                if meta.max_accounting.len() != accounting_map.len() {
```

Comment:
This will still override the contents, is that fine?

Reply:
it seems fine